public/maintenance.html

```diff
@@ -1,39 +1,55 @@
-
 <!DOCTYPE html>
-<html>
-<head>
-  <title>We are undergoing maintenance</title>
+<!-- saved from url=(0057)https://s3.amazonaws.com/thepracticaldev/maintenance.html -->
+<html><head><meta http-equiv="Content-Type" content="text/html; charset=windows-1252">
```

Comment:
`charset=windows-1252` is this needed? (I honestly don't know, but not using utf-8 sounds interesting)

Reply:
I actually think this is just some copypasta — no specific intentions here.

Reply:
Updated!